src/main/resources/com/blackducksoftware/integration/jira/i18n_6.properties

```diff
@@ -104,3 +104,19 @@ blackduck.integration.jira.server.details.proxy.host.description=The hostname of
 blackduck.integration.jira.server.details.proxy.port.description=The port of the Proxy to use, Integer
 blackduck.integration.jira.server.details.proxy.username.description=The Proxy username for authenticated Proxies
 blackduck.integration.jira.server.details.proxy.password.description=The Proxy password for authenticated Proxies
+# Migration fields
+blackduck.integration.jira.migration.banner.label=This tab will allow you to update the issue properties of the Black Duck issues created by this plugin so that Alert will recognize the issues and update them if needed.
+blackduck.integration.jira.migration.alert.link.label=Click here to check out Alert.
```

Comment:
I would change this to, "Click here for more information about Alert." The phrase, "check out" could be mistaken for the git concept.